code/modules/hydroponics/grown/banana.dm

```diff
@@ -151,7 +151,7 @@
 	spawn(30)
 		if(!QDELETED(src))
 			var/mob/living/simple_animal/banana_spider/S = new /mob/living/simple_animal/banana_spider(get_turf(src.loc))
-			S.speed += round(10 / seed.potency)
+			S.speed += round(10 - 10 * (seed.potency/100), 1)
```

Comment:
I know this is fixing a divide-by-zero, but it's a... totally different equation.
![image](https://user-images.githubusercontent.com/3927690/76472458-d5c4e280-63b2-11ea-9900-d8b8a75de179.png)